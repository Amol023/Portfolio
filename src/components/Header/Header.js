import React from "react";
import { css } from "emotion";
import styled from "react-emotion";

const StyledHeader = styled("div")`
  background: ${({ theme: { colors }}) => colors.green.teal};
  width: 100%;
  height: 5%;
  text-align: center;
`;

const Header = () => {
  return <StyledHeader />;
};

export default Header;
