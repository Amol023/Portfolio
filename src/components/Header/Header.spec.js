import React from 'react';
import * as emotion from 'emotion';
import { createSerializer } from 'jest-emotion';
import Header from './Header';
import { getTestComponent } from '../../../test/helpers';

expect.addSnapshotSerializer(createSerializer(emotion));

describe('<Header />', () => {
  it('renders without crashing', () => {
    const component = <Header />;
    expect(getTestComponent({ component, mount: true })).toMatchSnapshot();
  });
});
