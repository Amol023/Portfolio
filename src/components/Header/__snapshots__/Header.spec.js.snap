// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders without crashing 1`] = `
.emotion-0 {
  background: #009688;
  width: 100%;
  height: 5%;
  text-align: center;
}

<ThemeProvider
  theme={
    Object {
      "colors": Object {
        "black": Object {
          "jet": "#000000",
          "primary": "#222222",
        },
        "blue": Object {
          "light": "#6DA2B7",
        },
        "green": Object {
          "dark": "#0F5357",
          "teal": "#009688",
        },
        "grey": Object {
          "charcoal": "#404040",
          "dark": "#222222",
          "darkish": "#2E2E2E",
          "eclipse": "#373737",
          "faded": "#7A7A7A",
          "fadeout": "rgba(255, 255, 255, 0.95)",
          "hover": "#BBBBBB",
          "light": "#E5E5E5",
          "lightish": "#E9E9E9",
          "medDark": "#C4C4C4",
          "medium": "#DDDDDD",
          "semiDark": "#535353",
          "superlight": "#F0F0F0",
        },
      },
    }
  }
>
  <Header>
    <Styled(div)>
      <div
        className="emotion-0"
      />
    </Styled(div)>
  </Header>
</ThemeProvider>
`;
